core/src/main/java/org/elasticsearch/search/rescore/QueryRescorerBuilder.java

```diff
@@ -78,6 +75,25 @@ public QueryRescorerBuilder(QueryBuilder<?> builder) {
     }
 
     /**
+     * Read from a stream.
+     */
+    public QueryRescorerBuilder(StreamInput in) throws IOException {
+        super(in);
```

Comment:
seems a little risky... but we should have tests for serialization that catch these things!